clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -770,7 +772,8 @@ public void handle(OffsetFetchResponse response, RequestFuture<Map<TopicPartitio
                     log.debug("Group {} failed to fetch offset for partition {}: {}", groupId, tp, error.message());
 
                     if (error == Errors.UNKNOWN_TOPIC_OR_PARTITION) {
-                        future.raise(new KafkaException("Partition " + tp + " may not exist or user may not have Describe access to topic"));
+                        future.raise(new KafkaException("Partition " + tp + " may not exist or the user may not have " +
```

Comment:
FYI: I created https://issues.apache.org/jira/browse/KAFKA-4665 for the inconsistencies mentioned above.